pombola/south_africa/views.py

```diff
@@ -59,6 +59,10 @@
 from django_date_extensions.fields import ApproximateDate
 
 
+# For requests to external APIs, timeout after 3 seconds:
+API_REQUESTS_TIMEOUT = 3
```

Comment:
http://requests.readthedocs.io/en/master/user/advanced/#timeouts suggests (if the example is anything to go by?) 3.05, but meh :)